manifests/init.pp

```diff
@@ -160,6 +160,8 @@
   Enum['present', 'absent'] $ensure  = 'present',
 ) {
 
+  tag('pe_patching_only')
```

Comment:
I think `pe_patch_only` would align better with the module name?

Reply:
I think we should use something more obscure to prevent the possibility of name collisions as I think these are all in one global space? Like tagging on a uuid? [allowed characters for tags](https://www.puppet.com/docs/puppet/7/lang_reserved#tag-names)

Reply:
Interesting idea @steveax ; I *think* that this is not meant for any human-facing UI, so that using a UUID that is specified known by the patching sounds reasonable. But I am less inclined to do that if this is a user facing id...

Reply:
are people happy with the new name?

Reply:
Looks good to me Tony.  Thank you.